profiles/2017-03-09/definition.json

```diff
@@ -16,7 +16,7 @@
         "../../services/resources/mgmt/2016-09-01/links/linksapi",
         "../../services/resources/mgmt/2015-01-01/locks",
         "../../services/resources/mgmt/2015-01-01/locks/locksapi",
-        "../../services/resources/mgmt/2015-10-01-preview/policy",
+        "../../services/preview/resources/mgmt/2015-10-01-preview/policy",
         "../../services/resources/mgmt/2016-02-01/resources",
```

Comment:
hmm so the profile contains a preview api ?

Reply:
Sadly yes.

Reply:
Will fix per discussion to remove `preview` directory.